Engine/DataFeeds/TextSubscriptionDataSourceReader.cs

```diff
@@ -69,6 +83,8 @@ public TextSubscriptionDataSourceReader(IDataCacheProvider dataCacheProvider, Su
             _config = config;
             _isLiveMode = isLiveMode;
             _factory = (BaseData) ObjectActivator.GetActivator(config.Type).Invoke(new object[] { config.Type });
+            _shouldCacheDataPoints = !_config.IsCustomData && _config.Resolution >= Resolution.Hour
```

Comment:
Why not cache higher frequency data?

Reply:
Both `Resolution.Daily` and `Resolution.Hour` data files are organized in 1 zip file per symbol (for the entire data history). Higher frequency data (`Minute`, `Second`, `Tick`) are organized in daily files. So there is a big difference in potential performance improvement. So as an initial approach tackling `Resolution.Daily` and `Resolution.Hour` seems like a good idea, also having in mind the potential RAM usage too.